packages/loot-core/src/server/main.ts

```diff
@@ -148,13 +148,7 @@ handlers['transactions-filter'] = async function ({
   notPaged,
   options = {},
 }) {
-  return db.getTransactions(
-    term,
-    accountId,
-    latestDate,
-    notPaged ? null : count == null ? undefined : count,
-    options,
-  );
+  return db.getTransactions(accountId);
```

Comment:
Is `'transactions-filter'` even used? Couldn't find where, so perhaps worth removing entirely.